styles.css

```diff
@@ -684,3 +712,413 @@ footer {
+    width: 300px;
+    list-style: none;
+    padding-right: 30px;
+    align-items: center;
+    font-family: "Inter", sans-serif;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+  }
+
+  nav ul li a {
+    color: #fbf8f7;
+    text-decoration: none;
+  }
+
+  .top-header > div {
+    display: flex;
+    flex-direction: row;
+    position: absolute;
+    justify-content: space-between;
+    height: 72px;
+    left: 0;
+    top: 0;
+    z-index: 1;
+    background-color: #3c3a39;
+    width: 100%;
+    max-width: 1440px;
+  }
+
+  .recent-para {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+
+    /* text-align: left; */
+    width: 100%;
+  }
+
+  .logo-art {
+    width: 100%;
+    height: 20px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    padding-left: 227px;
+  }
+
+  #main {
+    background-color: #1c1a19;
+    color: white;
+    background-image: url(images/background.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: 34%;
+    width: 100%;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+  }
+
+  #main > div {
+    width: none;
+    max-width: 1440px;
+    margin-left: -5%;
+  }
+
+  h1 {
+    width: 100%;
+    height: 124px;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #ff6b00;
+    margin-top: 20px;
+  }
+
+  br {
+    display: inline;
+  }
+
+  #h2 {
+    width: 100%;
+    height: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
+
+  .data-arc {
+    width: 100%;
+    height: 72px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #f4f5f7;
+    margin-left: 20px;
+  }
+
+  .social-media {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 190px;
+    width: auto;
+    margin-right: 5%;
+    align-self: center;
+  }
+
+  .just-items {
+    padding: 0 120px 0;
+  }
+
+  .pic-ajust {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .work-story {
+    padding-top: 0;
+  }
+
+  /* --------------------about me section -------------- */
+
+  .about-me {
+    background: none;
+  }
+
+  .tech-skills {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 90px 0;
+  }
```

Comment:
- [ ] Please I kindly suggest sticky to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1278&t=m82U1CnKG1ogD0DD-0)

_**REQUIRED FIGMA DESIGN**_


![Screenshot (134)](https://user-images.githubusercontent.com/78104544/228998587-c634890e-c601-463f-88bd-fe1d718fed14.png)


_**YOUR DESIGN**_

- [ ] Please I suggest making changes to the `margin` of the `tech-skills` class. Instead of adding margins to only the top and bottom i.e `margin: 90px 0`, I suggest using `margin: 90px` to implement margins around the whole content. 😄 

![Screenshot (136)](https://user-images.githubusercontent.com/78104544/228998797-7e271465-736c-4f21-8437-6ec98875a775.png)

Reply:
resolved